docs/pages/build-reference/infrastructure.md

```diff
@@ -146,19 +146,20 @@ When selecting an image for the build you can use the full name provided below o
   - Every build gets its own fresh macOS VM
   - Hardware: Intel(R) Core(TM) i7-8700B CPU (6 cores/12 threads), 64 GB RAM
   - Build resource limits: 3 cores, 12 GB RAM
-- npm cache. [Learn more](caching/#javascript-dependencies)
+- NPM cache. [Learn more](caching/#javascript-dependencies)
```

Comment:
https://css-tricks.com/start-sentence-npm/#:~:text=Asking%20this%20question%20was%20a,precedent%20for%20not%20capitalizing%20it.

Reply:
We used capitalized "npm" in line 33 and I thought it was correct 😄 I will change it as well